transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py

```diff
@@ -196,6 +198,21 @@ def df_to_serviced_organisations(
         )
 
 
+def df_to_serviced_organisation_working_days(
+    df: pd.DataFrame, columns_to_drop: list, columns_to_drop_duplicates: list
+) -> Iterator[ServicedOrganisationWorkingDays]:
+    if not df.empty:
+        df_to_load = df.drop(columns=columns_to_drop)
+        for record in df_to_load.drop_duplicates(
```

Comment:
we can avoid creating copy of the dataframe by using .drop along with the drop_duplicates:-|

```
        for record in df.drop(columns=columns_to_drop).drop_duplicates(
            subset=columns_to_drop_duplicates
        ).itertuples(index=False)
```

Reply:
wanted to avoid inplace edit as the dataframe used is serviced_organisation which can be used by another function